source/Xunit.Gherkin.Quick/Scenario/ScenarioXUnitTestCaseRunner.cs

```diff
@@ -7,9 +7,9 @@
 
 namespace Xunit.Gherkin.Quick
 {
-    internal sealed class ScenarioXunitTestCaseRunner : XunitTestCaseRunner
+    internal sealed class ScenarioXUnitTestCaseRunner : XunitTestCaseRunner
```

Comment:
Just to be clear, correct spelling is Xunit, not XUnit. U should be lower case.
Please fix accordingly.